cluster/saltbase/salt/e2e-image-puller/e2e-image-puller.manifest

```diff
@@ -35,6 +35,14 @@ spec:
       name: socket
     - mountPath: /usr/bin/docker
       name: docker
+  # Add a container that runs a health-check
+  - name: nethealth-check
+    resources:
+      requests:
+        cpu: 100m
```

Comment:
I guess you're setting 100m 100m to get guaranteed qos, this is what ia was asking for confirmation about (that this 100m won't eject any other pods)